flow/e2e/postgres/qrep_flow_pg_test.go

```diff
@@ -166,7 +166,7 @@ func (s PeerFlowE2ETestSuitePG) TestSimpleSlotCreation() {
 		setupError <- s.conn.SetupReplication(context.Background(), signal, setupReplicationInput)
 	}()
 
-	s.t.Log("waiting for slot creation to complete: ", flowJobName)
+	s.t.Log("waiting for slot creation to complete: " + flowJobName)
```

Comment:
can use `slog.String`?

Reply:
s.t.Log isn't slog, this is testing. Moved to concat since it was a false positive on my scan